docs-web/src/main/java/com/sismics/docs/rest/resource/MessageResource.java

```diff
@@ -91,10 +91,12 @@ public Response list(@QueryParam("sort") Integer sortColumn, @QueryParam("asc")
      * @apiError (client) ForbiddenError Access denied
      * @apiPermission user
      * @apiVersion 1.5.0
+     * 
+     * @param count Current number of unread messages
      */
     @GET
     @Path("unread_count")
-    public void getUnreadCount(@Suspended final AsyncResponse asyncResponse)
+    public void getUnreadCount(@QueryParam("count") Integer count, @Suspended final AsyncResponse asyncResponse)
```

Comment:
What's the reason for sending an additional count parameter?

Reply:
We are keeping track of the count so that if the message count remains the same, we wouldn't create a new JsonObject for the same count.